infra/cifuzz/cifuzz.py

```diff
@@ -460,3 +481,27 @@ def parse_fuzzer_output(fuzzer_output, out_dir):
   summary_file_path = os.path.join(out_dir, 'bug_summary.txt')
   with open(summary_file_path, 'a') as summary_handle:
     summary_handle.write(summary_str)
+
+
+def is_project_sanitizer(sanitizer, oss_fuzz_project_name):
+  """Finds all of the sanitizers a project can use for building and running.
+
+  Args:
+   sanitizer: The desired sanitizer.
+   oss_fuzz_project_name: The name of the relevant OSS-Fuzz project.
+
+  Returns:
+  True if project can use sanitizer.
+  """
+  project_yaml = os.path.join(helper.OSSFUZZ_DIR, 'projects',
```

Comment:
Could you change this variable to `OSS_FUZZ_DIR`?

Reply:
I will do this in a separate PR because It occurs and many more places than CIFuzz.

Reply:
Landed with #3535